packages/core/bundle/fonts.css

```
@@ -0,0 +1 @@
+@import url("https://rsms.me/inter/inter.css");
```

Stylelint (17.14.1)

[error] Expected "url("https://rsms.me/inter/inter.css")" to be ""https://rsms.me/inter/inter.css"" (import-notation)

(import-notation)